plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
In case of MTR tests  - no, this was what I was referring to. In case of this application - yes it should be possible, however do we want to do this, since Vault used in MTR needs cleaning anyways ? I would need to keep a hash map for each key for which delete failed - this means additional mutex or using lock free hash map. The later would be better as the main goal of this test is to check if there are no conflicts/deadlocks when multiple threads are run and any serialization is not good.